ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-LEAF2B.yml

```diff
@@ -888,6 +880,15 @@ vlan_interfaces:
     ip_address_virtual: 10.4.12.254/24
     ipv6_address_virtual: 2001:db8:412::1/64
     mtu: 1560
+ipv6_static_routes:
+- destination_address_prefix: ::/0
+  vrf: Tenant_D_OP_Zone
+  gateway: 2001:db8:311::4
+  name: IPv6-test-2
+- destination_address_prefix: 10.3.11.0/24
```

Comment:
This v6 route has a v4 destination prefix, is this intentional?

Reply:
Good spot, I pushed the changes! Thank you.